common/c_cpp/src/c/mempool.c

```diff
@@ -76,6 +76,7 @@ memoryNode*
 memoryPool_addNode (memoryPool* pool, size_t nodeSize)
```

Comment:
I build with scons on windows, let me check if there is a compiler arg I can use to allow this, then I can remove all of these changes.